File: Hishab Server/hishabApp/hishabApp/src/main/java/com/example/hishab/hishabApp/apiModels/ShoppingItemResponse.java
package com.example.hishab.hishabApp.apiModels;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class ShoppingItemResponse {
	private int localId;
	private int shoppingItemId;
	private ProductResponse productResponse;
}

File: Hishab/app/src/main/java/com/example/hishab/retrofit/response/ShoppingItemResponse.kt
package com.example.hishab.retrofit.response

import com.example.hishab.retrofit.response.ProductResponse

class ShoppingItemResponse {
    var localId = 0
    var shoppingItemId = 0
    var productResponse: ProductResponse? = null

    constructor() {}
    constructor(localId: Int, shoppingItemId: Int, productResponse: ProductResponse?) {
        this.localId = localId
        this.shoppingItemId = shoppingItemId
        this.productResponse = productResponse
    }
}